content/zh/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -945,7 +945,7 @@ node selector.
 2. If the namespace lacks such an annotation, use the `clusterDefaultNodeSelector` defined in the `PodNodeSelector`
 plugin configuration file as the node selector.
 3. Evaluate the pod's node selector against the namespace node selector for conflicts. Conflicts result in rejection.
-4. Evaluate the pod's node selector against the namespace-specific whitelist defined the plugin configuration file.
+4. Evaluate the pod's node selector against the namespace-specific allowlist defined the plugin configuration file.
```

Comment:
Hello @scottstout .
I worry about _global_ changes to the content. Have you discussed your ideas with the SIG-Docs group?
https://kubernetes.io/docs/contribute/#get-involved-with-sig-docs

Reply:
@kbhawkey I haven't discussed with them.

Reply:
@kbhawkey are you just suggesting we roll back the changes under content/zh ?

Reply:
Hi @scottstout, It would be helpful if you limited content to a single localization.

Reply:
Done